eng/versioning/version_client.txt

```diff
@@ -22,7 +22,7 @@ com.azure:azure-e2e;1.0.0-beta.1;1.0.0-beta.1
 com.azure:azure-identity;1.0.4;1.1.0-beta.3
 com.azure:azure-messaging-eventhubs;5.0.2;5.1.0-beta.1
 com.azure:azure-messaging-eventhubs-checkpointstore-blob;1.0.2;1.1.0-beta.1
-com.azure:azure-messaging-servicebus;1.0.0-beta.1;1.0.0-beta.1
+com.azure:azure-messaging-servicebus;7.0.0-beta.1;7.0.0-beta.1
```

Comment:
Just so everyone else is aware. This version isn't arbitrary, all 4 language that are releasing similar libraries have decided to start with the version of 7.